### PR TITLE
Align operation to flow structure

### PR DESCRIPTION
**Pull request by oritstone**

### CODE REVIEW

**Comment by oritstone** [comment]
@orius123 please review & merge (only after you merge the matching pull request in score-language)

File: org/openscore/slang/base/comparisons/less_than_percentage.sl [hunk 1]
```diff
@@ -17,22 +17,34 @@
...
-          - SUCCESS: first_percentage_nr < second_percentage_nr
-          - FAILURE
+operation:
+  name: less_than_percentage
+  inputs:
+    - first_percentage
+    - second_percentage
+  action:
+    python_script: |
+      error_message = ""
```
Comment (orius123) [inline org/openscore/slang/base/comparisons/less_than_percentage.sl]: is this change related to the operation new format??
